kit/foundation/docs/template/docs/foundation/concepts.md

```diff
@@ -0,0 +1,8 @@
+# Concepts
+
+## Landing Zone
```

Comment:
we can link to CFMM here, though I'm not sure there's a single page that explains LZ, e.g. in the CFMM Knowledge Hub.

That probably needs a bit of a makeover as well https://cloudfoundation.org/understanding-cloud-foundation/#why-build-a-cloud-foundation

Reply:
(though that's out of scope for collie-hub here today)